aiida_pseudo/groups/family/pseudo.py

```diff
@@ -150,6 +151,85 @@ def create_from_folder(cls, dirpath, label, description='', deduplicate=True):
 
         return family
 
+    @classmethod
+    def validate_cutoffs(cls, elements: set, cutoffs: dict) -> None:
```

Comment:
I originally explicitly did not implement this on the base class, because I was not sure that all pseudo types would 1) need recommended cutoffs like this (the concept might not even make sense) and 2) might require a different implementation/definition. Now that there is a second one, the pseudo dojo pseudos, that seem to be able to use the same concept, I agree that it may seem logical to move this here, but this may not always be the case. Of course then we could create another subclass that implements this, but this may cause trouble with data migrations. Maybe we should already consider doing this now?

Reply:
I did this just because it was the shortest path to me, and I didn't really know what I was doing. It definitely makes sense to implement the cutoff suggestions in a subclass and then go from there if we expect some types of pseudos to never need this feature.